common/models/access-token.js

```diff
@@ -146,6 +146,16 @@ module.exports = function(AccessToken) {
       var userRelation = AccessToken.relations.user; // may not be set up
       var User = userRelation && userRelation.modelTo;
```

Comment:
> @bajtos : I think we need to honour AccessToken.relations.user.modelTo as the default User model. Consider the case where app has custom User and AccessToken models, e.g. in order to customise some of built-in methods and/or settings.
In that case, principalType is not set. Before your change, the code will look up the related user model. After your change, the built-in User model would be incorrectly used.

i moved back to the original implementation following your comment.

**although**: if I correctly get the [docs](https://apidocs.strongloop.com/loopback/v/3.2.1/#loopback-getmodelbytype) for `Registry.getModelByType(...)` we could use `getModelByType('User')` instead of `AccessToken.relations.user.modelTo` as the method allows to 
> to find configured models in models.json **over the base model**.